aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -130,6 +132,52 @@ The following examples use the <xref:Microsoft.AspNetCore.Server.Kestrel.Core> n
 using Microsoft.AspNetCore.Server.Kestrel.Core;
 ```
 
+Kestrel options, which are configured in C# code in the following examples, can also be set using a [configuration provider](xref:fundamentals/configuration/index). For example, the File Configuration Provider can load Kestrel configuration from a file (*appsettings.json* or *appsettings.{Environment}.json*):
+
+```json
+{
+  "Kestrel": {
+    "Limits": {
+      "MaxConcurrentConnections": 100,
+      "MaxConcurrentUpgradedConnections": 100
+    },
+    "DisableStringReuse": true
+  }
+}
+```
+
+Use **one** of the following approaches:
+
+* Inject an instance of `IConfiguration` into the `Startup` class from the `Microsoft.Extensions.Configuration` namespace (`Configuration` in the following example). In `Startup.ConfigureServices`, load the `Kestrel` section of configuration into Kestrel's configuration:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    services.Configure<KestrelServerOptions>(
+        Configuration.GetSection("Kestrel"));
+}
+```
+
+* In the `Program` class (*Program.cs*), load the `Kestrel` section of configuration into Kestrel's configuration:
```

Comment:
In *Program.cs*, load the `Kestrel` section of configuration into Kestrel's configuration: